src/limits/TimeLimitListener.cc

```diff
@@ -88,24 +88,27 @@ executor::ExecuteAction TimeLimitListener::onSigalrmSignal() {
     if (!isTimerCreated_) {
         return executor::ExecuteAction::CONTINUE;
     }
-    return verifyTimeUsage();
+    auto time = getTimeUsage();
+    return verifyTimeUsage(move(time));
 }
 
 void TimeLimitListener::onPostExecute() {
     // TODO: run this just after child exit
-    verifyTimeUsage();
-    // TODO Save time usage to OutputBuilder.
+    auto time = getTimeUsage();
+    outputBuilder_->setRealTimeMicroseconds(time->realTimeUs);
```

Comment:
I think we should double-check this, but IIRC the old "real-cpu" supervisor uses user time, not real time.